Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -339,14 +339,21 @@ def fetch_notables(service: client.Service, mapper: UserMappingObject, comment_t
     latest_time = last_run_latest_time or now
     kwargs_oneshot = build_fetch_kwargs(params, occured_start_time, latest_time, search_offset)
     fetch_query = build_fetch_query(params)
+    last_run_fetched_ids = last_run_data.get('found_incidents_ids', {})
+    if late_indexed_pagination := last_run_data.get('late_indexed_pagination'):
+        # code for additional pagination fetch here
+        window = f'{kwargs_oneshot.get("earliest_time")}-{kwargs_oneshot.get("latest_time")}'
+        demisto.debug(f'[SplunkPy] additional fetch for the window {window} to check for late indexed incidents')
+        if last_run_fetched_ids:
+            exclude_id_where = f'where not event_id in ({",".join(last_run_fetched_ids)})'
```

Comment:
Correct, that is why we have it is: **",".join(last_run_fetched_ids)**

Reply:
no :)
`",".join(last_run_fetched_ids)` will result `(id_1, id_2)`
otherwise how the test here passed ?

<img width="733" alt="image" src="https://github.com/demisto/content/assets/79846863/c96287ea-7a50-4e4a-b876-efb441206cca">

Reply:
True, I fixed it